swift-executor.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
   #
 
   s.name         = "swift-executor"
-  s.version      = "1.8.6"
```

Comment:
We're using this version change in the Paperless Post fork.  Feel free to ignore it.

Reply:
@dornad thanks for this, I'll merge it in